modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration-custom-resource-properties_{context}"]
+= NotificationsConfiguration custom resource properties
+
+The `NotificationsConfiguration` resource is a Kubernetes custom resource (CR) that manages notifications in a Kubernetes cluster. In {gitops-title}, you can add templates, triggers, services, and subscription resources to an Argo CD `Notifications` config map by using  the `NotificationsConfiguration` CR.
+
+When you create a cluster in {gitops-title} with notifications enabled, a `NotificationsConfiguration` CR is created by default with the name `default-notifications-configuration`.
+
+Any change made in the existing configuration of the `NotificationsConfiguration` CR is replicated in the Argo CD `Notifications` config map. For example, if the user adds  trigger configuration in the `NotificationsConfiguration` resource, this configuration is read, processed, and updated in the Argo CD `Notifications` config map.
+
+[IMPORTANT]
+====
+Any configuration changes must be updated in  the `default-notifications-configuration` CR. Custom resources created by the users for `NotificationsConfiguration` resource are not supported.
+
+Any modification to the Argo CD `argocd-notifications-cm` config map is overridden by the changes made in the `NotificationsConfiguration` CR.
+====
+
+.`NotificationsConfiguration` custom resource properties
+[cols="1,1,1",options="header"]
+|====
+|Properties |Description|Default value
+
+|`templates`
+|Templates are used to generate the notification template message.
+|empty
+
+|`triggers`
+|Triggers are used to define the condition when a notification is sent to the user and the list of templates required to generate the message.
+|empty
+
+|`services`
+|Services are used to deliver a message.
+|empty
+
+|`subscriptions`
```

Comment:
@Dhruv-Soni11 pls check whish subscriptions you are mentioning here. The one with capital S or with small case. Consult with SME if need be.

Also. The table format is much better aiding in better readability. Pls note that in previous table default is the second column. Maintain consistency please. Also see how empty is written in previous tables. Consult with SME on this if need be and do the needful Thank you.

Reply:
I have just fixed this feedback, @Srivaralakshmi. Please check and confirm if it is fine.

Reply:
@Dhruv-Soni11 Please pay attention to the comments given. You have not implemented it fully. Move Default to the 3rd column. TY!

Reply:
@Srivaralakshmi - I did pay attention and changed the **default** column placement from the 3rd column to the 2nd column by referring to the table in the Repo Server Properties section. I believe your previous statement said the **default** column is in the 2nd column. I am happy to change it again.

However, I detect an inconsistency with the **default**" column placement in the previous sections.

In the Repo Server Properties section, the **default** column is the second column, and

In the Argo CD custom resource properties section, the **default** column is the third column.

Do you suggest that we remove the inconsistency from there as well?

Reply:
@Dhruv-Soni11 Yes, what I mean is exactly that. We must give our readers/users a unified experience. Thanks for the understanding. I really do appreciate the effective change you are bringing into the GitOps docs through your great work. Thanks a lot.

Reply:
Hi @Srivaralakshmi - As per the [document](https://docs.google.com/document/d/16wc3zqke03RDFAm1zv0Qjdo9lL2MgQE2orA1HmoHoeo/edit#heading=h.3qhjblbib0f8) shared by @iam-veeramalla, the default column is the second column. PTAL.